chart/templates/server-ide-configmap.yaml

```diff
@@ -56,6 +56,7 @@ options:
     orderKey: "02"
     title: "VS Code"
     type: "desktop"
+    label: "Desktop"
```

Comment:
**thought**: Here's the [relevant discussion](https://github.com/gitpod-io/gitpod/pull/8783#discussion_r828513588) and consensus reached to drop the label here. Cc @mustard-mh @loujaybee

Reply:
I prefer to add something to make desktop and browser one look different. cc @loujaybee